src/client/src/orangehrmRecruitmentPlugin/pages/AddJobVacancy.vue

```diff
@@ -78,17 +78,17 @@
         </oxd-grid>
         <oxd-grid :cols="3" class="orangehrm-full-width-grid">
           <oxd-grid-item class="orangerhrm-switch-wrapper">
-            <oxd-text class="orangehrm-text" tag="p">{{
-              $t('recruitment.active')
-            }}</oxd-text>
+            <oxd-text class="orangehrm-text" tag="p">
+              {{ $t('recruitment.active') }}
+            </oxd-text>
             <oxd-switch-input v-model="vacancy.status" />
           </oxd-grid-item>
         </oxd-grid>
         <br />
         <oxd-grid :cols="3" class="orangehrm-full-width-grid">
           <oxd-grid-item class="orangerhrm-switch-wrapper">
             <oxd-text class="orangehrm-text" tag="p">
-              {{ $t('publish _in_rss_feed _and_web_page') }}
+              {{ $t('recruitment.publish_in_rss_feed _and_web_page') }}
```

Comment:
```suggestion
              {{ $t('') }}
```
wrong
`recruitment.publish_in_rss_feed[SPACE]_and_web_page`

check in `src/client/src/orangehrmRecruitmentPlugin/pages/EditJobVacancy.vue`

Reply:
This is my mistake